processor/src/test/java/org/mapstruct/ap/test/value/SpecialOrderMapper.java

```diff
@@ -0,0 +1,48 @@
+/**
+ *  Copyright 2012-2016 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.value;
+
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.ValueMapping;
+import org.mapstruct.ValueMappingType;
+import org.mapstruct.ValueMappings;
+import org.mapstruct.factory.Mappers;
+
+/**
+ * @author Sjaak Derksen
+ */
+@Mapper
+public interface SpecialOrderMapper {
+
+    SpecialOrderMapper INSTANCE = Mappers.getMapper( SpecialOrderMapper.class );
+
+    OrderDto orderEntityToDto(OrderEntity order);
+
+    @ValueMappings({
+        @ValueMapping( target = "DEFAULT", valueMappingType = ValueMappingType.NULL ),
```

Comment:
Yes, we can do a quick call next week. How about Monday?

> we leave room open for enum-to-string and string-to-enum

Yes, that's true. Actually, `@ValueMapping` is growing on me if we go for the more concise "null" / "\*" syntax. 

Of course these are valid strings, but this could be handled using escaping. So if you really wanted to map to the Strings "\*" or "null", you'd have to put some escaping in place: "\\\*" and "\null". If you wanted to map into "\\\*", the backslash would have to be escaped as well: "\\\\*" and so on. We also could use "\_null\_" instead of "null", decreasing the number of cases where escaping is needed.

Bottom line being, if we managed to get rid of `ValueMappingType` that way, I'd like it much better :)

How does this sound?